redisenterprise/datadog_checks/redisenterprise/data/conf.yaml.example

```diff
@@ -292,8 +297,8 @@ instances:
     ## @param tls_ca_cert - string - optional
     ## The path to a file of concatenated CA certificates in PEM format or a directory
     ## containing several CA certificates in PEM format. If a directory, the directory
-    ## must have been processed using the c_rehash utility supplied with OpenSSL. See:
-    ## https://www.openssl.org/docs/manmaster/man3/SSL_CTX_load_verify_locations.html
+    ## must have been processed using the `openssl rehash` command. See:
+    ## https://www.openssl.org/docs/man3.2/man1/c_rehash.html
```

Comment:
Can you provide insight on these changes? We will need another review from agent eng since this was done after their approval. I will request a new review now!